src/Rings/mpolyquo-localizations.jl

```diff
@@ -1966,3 +1966,97 @@ function (f::Oscar.MPolyAnyMap{<:MPolyRing, <:MPolyQuoLocRing, <:MPolyQuoLocaliz
   g = get_attribute(f, :lifted_map)
   return codomain(f)(g(a), check=false)
 end
+
+function vector_space(kk::Field, W::MPolyQuoLocRing;
+    ordering::MonomialOrdering=degrevlex(gens(base_ring(W)))
```

Comment:
If you are in MPolyQuoLocRing, then this should not be the default. Maybe negdegrevlex ?

Reply:
No. This is correct.